src/index.html

```diff
@@ -1,13 +1,354 @@
+                historical perspective.
+              </p>
+            </article>
+            <article class="product explore-online__product">
+              <a href="#" class="product__link">
+                <img class="product__photo"
+                     src="images/Layer 53@1,5x.png"
+                     alt="Virtual group tours"
+                >
+              </a>
+              <h3 class="product__title">Travel</h3>
+              <p class="product__description">
+                Request a guided tour, reserve admission
+                and lead your own small group, or request a
+                live virtual tour led by a Museum guide.
+              </p>
+            </article>
+          </div>
+        </section>
+        <section class="contact-us" id="contact-us">
+          <div class="phone__bar contact-us__bottom">
+          <h2 class="section-title section-title-center">Contact us</h2>
+            <div class="contact-us__locations">
+              <div class="contact-us__locations__left">
+                <h4 class="contact-us__location">The Met Fifth Avenue</h4>
+                <p class="contact-us__phone-title">Phone</p>
+                <a
+                  class="contact-us__phone-number"
+                  href="tel:+1 212-535-7710"
+                >
+                  +1&nbsp;212-535-7710
+                </a>
+                <p class="contact-us__address-title">Address</p>
+                <a target="_blank"
+                   href="https://maps.app.goo.gl/wC2af3wikRVnYVSY9"
+                   class="contact-us__address-number">
+                     1000 Fifth Avenue
+                     New York, NY 10028
+                </a>
+              </div>
+              <div class="contact-us__locations__right">
+                <h4 class="contact-us__location">The Met Cloisters</h4>
+                <p class="contact-us__phone-title">Phone</p>
+                <a
+                  class="contact-us__phone-number"
+                  href="tel:+1 212-923-3700"
+                >
+                  +1&nbsp;212-923-3700
+                </a>
+                <p class="contact-us__address-title">Address</p>
+                <a target="_blank"
+                   href="https://maps.app.goo.gl/EDoajftnc2erbxZi6"
+                   class="contact-us__address-number">
+                      99 Margaret Corbin Drive
+                      Fort Tryon Park
+                      New York, NY 10040
+                </a>
+              </div>
+            </div>
+            <div class="contact-us__form">
+              <form onsubmit="this.reset(); return false;">
+                <input
+                  class="contact-us__input contact-us__name"
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  required
+                >
+                <input
+                  class="contact-us__input contact-us__email"
+                  type="text"
```

Comment:
```suggestion
                  type="email"
```

And remove your pattern because it doesn't work
![image](https://github.com/mate-academy/layout_landing-page/assets/80700247/ceb09833-8fcd-4701-8ef1-07898917e88a)